client/analytics/components/report-table/index.js

```diff
@@ -0,0 +1,138 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import { withSelect } from '@wordpress/data';
+import { get, orderBy } from 'lodash';
+import PropTypes from 'prop-types';
+
+/**
+ * WooCommerce dependencies
+ */
+import { TableCard } from '@woocommerce/components';
+import { onQueryChange } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import ReportError from 'analytics/components/report-error';
+import { getReportChartData, getReportTableData } from 'store/reports/utils';
+
+class ReportTable extends Component {
+	render() {
+		const {
+			getHeadersContent,
+			getRowsContent,
+			getSummary,
+			itemIdField,
+			primaryData,
+			tableData,
+			totalsCountField,
+			// These two props are not used in the render function, but are destructured
+			// so they are not included in the `props` variable.
```

Comment:
Does it matter?

Reply:
I wanted to add the comment to make sure in the future we remember we are defining those variables in order to exclude them from the `tableProps` object. Otherwise, doing a cleanup of the code we might remove `endpoint` and `tableQuery` from the destructuring thinking defining them was a mistake.

That would not cause an error, so we would not notice it, but I think it's better not to pass unnecessary props down to `<TableCard>`.

But I don't have a strong opinion here, so I would be happy removing the comment if you think it's clear without it.

Reply:
>Otherwise, doing a cleanup of the code we might remove endpoint and tableQuery from the destructuring thinking defining them was a mistake.

Thats a good idea. Thanks for the clarification